src/modules/person/api/person.controller.spec.ts

```diff
@@ -85,4 +96,58 @@ describe('PersonController', () => {
             expect(personUcMock.findPersonById).toHaveBeenCalledTimes(1);
         });
     });
+
+    describe('when getting all persons', () => {
```

Comment:
The description should just be 'findAll'.
We don't need a "when"-sentence for this.